### ISSUE TITLE
clarify documentation regarding data sources / data stores

### ISSUE THREAD

**Issue by duritong** [choria-io/mcollective-choria#178]
Reading the documentation about [data sources](http://choria.io/docs/playbooks/data/) is a bit confusing to me:

* title is data sources
* the first example shows a yaml section `data sources`
* from the second section doc mainly talks about data stores and also shows usually a yaml section `data stores`
* The environment section more down has `data_stores` in its example.

Somehow I'm now not sure how to configure what in a playbook.

Also it would be nice if there is a complete example, including data source definitions. Maybe it's enough to add a `data source` (or `data store`...) section to the "Editing Data Sources" chapter.

**Comment by ripienaar** [comment]
Hehe yeah I noticed the data source and data store confusions too. Will try clarify. I need to stick up a few full examples in a directory rather than the docs I suspect. People expect to be able to run doc examples verbatim 

Will see what I can do this week. For sure it's confusing

**Comment by ripienaar** [comment]
tl;dr s/store/source

The docs have been updated to use the right terms and so now - its Data Store and Data Stores.  I think `Sources` would have been a better word, too late now :)

Let me know what you think, I'll look at adding full example playbooks somewhere

**Comment by ripienaar** [comment]
closing this one, please re-open if unclear

**Comment by duritong** [comment]
The environment example still has `data_stores:` as a key. Otherwise it is now consistent.

(can't reopen the issue)

**Comment by ripienaar** [comment]
@duritong you mean here http://choria.io/docs/playbooks/data/#environment-store they're all `data_stores:` which is correct I think?

**Comment by duritong** [comment]
Ok, my issue. All this fancy new html doesn't render properly if you don't allow everything. Sorry for the noise...

![image](https://cloud.githubusercontent.com/assets/9511/22907933/eb83e020-f24b-11e6-8e5b-37c283bbea13.png)

But it's fine in chrome:

![image](https://cloud.githubusercontent.com/assets/9511/22907959/12ec99ea-f24c-11e6-89db-1c1ddfcd31f3.png)

**Comment by ripienaar** [comment]
@duritong lol, no problem, yeah this theme uses a client side syntax highlighter, not my favorite but I suck at html so this is what I got :)
